app/src/main/java/org/fossasia/openevent/app/common/di/module/NetworkModule.java

```diff
@@ -61,17 +80,18 @@ Picasso providesPicasso(Context context, OkHttpClient client) {
 
     @Provides
     @Singleton
-    Retrofit.Builder providesRetrofitBuilder(CallAdapter.Factory callAdapterFactory, Converter.Factory converterFactory, OkHttpClient client) {
```

Comment:
This was kept `Retrofit.builder` intentionally while implementing HostSelectionInterceptor. I think it was not working with supplying Retrofit directly to EventService Provider.